src/karthik/group-people-data/groupPeopleData.ts

```diff
@@ -0,0 +1,41 @@
+interface People {
+  _id: string;
+  name: string;
+  gender: string;
+  email: string;
+  phone: string;
+  city: string;
+}
+
+interface GroupOfPeopleByCity {
+  count: number;
+  city: string;
+  people: Array<People>;
+}
+
+const groupPeopleByCity = (
+  arrayOfObject: Array<People>
```

Comment:
Instead of arrayOfObject, please use some more meaningful name.

Reply:
clarified.